### ISSUE TITLE
Reassess how we make accessible radio selects into buttons

### ISSUE THREAD

**Issue by bryancasler** [4site-interactive-studios/engrid#165]
Cornershop notes

Ensuring forms can be navigated using only a keyboard is imperative for accessibility. Often, we see radio buttons that are transformed into larger, more visually-appealing buttons like this:

![radio selects as buttons](https://www.engagingnetworks.support/wp-content/uploads/2022/04/DonateButtons.png)

However, these buttons are not actually the radio inputs, but rather their associated labels. Labels are not accessible to keyboards in the same way that inputs are, and cannot be selected with either the spacebar or enter/return key. In fact, the radio inputs themselves are virtually always hidden using CSS. Screen readers see that, and skip them because of it.

Instead of using display or visibility to hide the input element, we have:

- Absolutely positioned it behind the label, out of sight
- Moved the input far outside the range of visibility, but without reducing opacity
- Created label styles that rely on the input selection to apply focus to the label as though it were the input

Now, these buttons can be navigated with the keyboard while remaining visually appealing!

**Comment by bryancasler** [comment]
Our checkboxes seem to be good enough, and we have more aria tools under our belt now. Closing this out.
